### DISCUSSION TITLE
Welcome to Wishing-Plan Discussions!

### DISCUSSION THREAD

**Discussion by Andreasgdp** [Andreasgdp/Wishing-Plan#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Andreasgdp** [comment]
Hi everyone, I am the owner and lead developer of Wishing Plan. My goal is to create a platform that helps people track their savings and plan for their future purchases, while also providing a space for community members to connect and share their wish lists with each other. I am excited to be a part of this community and to see what we can all achieve together.

**Comment by sjsanc** [comment]
Hey @Andreasgdp, I like the project and I'd like to help out. Is there an issue ticket you'd recommend I pick up? I like the look of #51 if you're ok with that. :)

**Reply by Andreasgdp** [reply]
All help is welcome :) Just be aware that we don't have the functionality to upload images at the moment (#43).

You are welcome to look into that as well; however, I think it is a good idea to keep it simple and start with #51 as you suggested.

**Reply by Andreasgdp** [reply]
@sjsanc If you leave a comment on the issues you want to work on, we will be able to assign you to them to let others know what is being worked on.
